src/index.js

```diff
@@ -15,29 +14,67 @@ const choices = [
   "Create a LinkFree JSON file",
   "Update an existing JSON file",
   "Provide a testimonial to a LinkFree user",
-  "Add a given testimonial to your JSON file",
```

Comment:
The reason why I `Add a given testimonial to your JSON file`. People who got testimonials from other people can add their usernames via CLI. I hope you are getting my point

<img width="225" alt="Screenshot 2023-02-17 at 12 33 55 AM" src="https://user-images.githubusercontent.com/51878265/219462640-9ad0e897-1249-41c7-81df-239fbfc327dd.png">

Reply:
> The reason why I add `Add a given testimonial to your JSON file`. People who got testimonials from other people can add their usernames via CLI. I hope you are getting my point
> 
> Some like this
> 
> <img width="225" alt="Screenshot 2023-02-17 at 12 33 55 AM" src="https://user-images.githubusercontent.com/51878265/219462640-9ad0e897-1249-41c7-81df-239fbfc327dd.png">
> 

I have kept the option to add testimonials to one's own profile look at the questions file

Reply:
Hey, 

Yes, I saw that. I think you didn't get my point. If a person has no linkfree profile and they are creating a new one. How they have already got some testimonials? I think I am not explaining well 😅

Reply:
Ok got it so let's remove it from the create option and just keep it in the update profile option will that be OK?

Reply:
Thank you. Yes, I update it makes sense.